CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+# [v1.2.0] (Unreleased)
+### Notes
+This release supports API1200 minimally where we can test OneView API1200 with this SDK. No new fields are added/deleted to support API1200. Complete support will be done in next releases.
```

Comment:
`use` API1200 would be more apt than `test` API1200. Dont you think?

Reply:
Modified